Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -2479,6 +2479,75 @@ bool ParseV2000CTAB(std::istream *inStream, unsigned int &line, RWMol *mol,
   return fileComplete;
 }
 
+void finishMolProcessing(RWMol *res, bool chiralityPossible, bool sanitize,
```

Comment:
Shouldn't this be named finishMDLMolProcessing?  Will it be used elsewhere?

Reply:
@greglandrum - a small comment here for a future thought.

Reply:
Sorry, I had done a review and then forgot to close it out.
This is only used in the mol file parser and will only be used there.
The naming isn't great, but it's consistent with the rest of the functions here.